src/index.html

```diff
@@ -1,43 +1,44 @@
-    <link rel="apple-touch-icon" sizes="76x76" href="./img/components/icons/favicon-apple-76x76.png">
-    <link rel="apple-touch-icon" sizes="120x120" href="./img/components/icons/favicon-apple-120x120.png">
-    <link rel="apple-touch-icon" sizes="152x152" href="./img/components/icons/favicon-apple-152x152.png">
-    <link rel="apple-touch-icon-precomposed" sizes="60x60" href="./img/components/icons/favicon-apple-60x60.png">
-    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="./img/components/icons/favicon-apple-76x76.png">
-    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="./img/components/icons/favicon-apple-120x120.png">
-    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="./img/components/icons/favicon-apple-152x152.png">
-    <script type="text/javascript">(function () {var PAGE_LOADED_TIME = Date.now(); window.getPageLoadedTime = function () {return PAGE_LOADED_TIME;}})();</script>
-    <!--[if BOOTSTRAP-CSS]><![endif]-->
-  </head>
+  <link rel="apple-touch-icon" href="./img/components/icons/favicon-apple-57x57.png">
+  <link rel="apple-touch-icon" sizes="72x72" href="./img/components/icons/favicon-apple-72x72.png">
+  <link rel="apple-touch-icon" sizes="114x114" href="./img/components/icons/favicon-apple-114x114.png">
+  <link rel="apple-touch-icon" sizes="144x144" href="./img/components/icons/favicon-apple-144x144.png">
+  <link rel="apple-touch-icon-precomposed" href="./img/components/icons/favicon-apple-57x57.png">
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="./img/components/icons/favicon-apple-72x72.png">
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="./img/components/icons/favicon-apple-114x114.png">
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="./img/components/icons/favicon-apple-144x144.png">
+  <link rel="apple-touch-icon" sizes="60x60" href="./img/components/icons/favicon-apple-60x60.png">
+  <link rel="apple-touch-icon" sizes="76x76" href="./img/components/icons/favicon-apple-76x76.png">
+  <link rel="apple-touch-icon" sizes="120x120" href="./img/components/icons/favicon-apple-120x120.png">
+  <link rel="apple-touch-icon" sizes="152x152" href="./img/components/icons/favicon-apple-152x152.png">
+  <link rel="apple-touch-icon-precomposed" sizes="60x60" href="./img/components/icons/favicon-apple-60x60.png">
+  <link rel="apple-touch-icon-precomposed" sizes="76x76" href="./img/components/icons/favicon-apple-76x76.png">
+  <link rel="apple-touch-icon-precomposed" sizes="120x120" href="./img/components/icons/favicon-apple-120x120.png">
+  <link rel="apple-touch-icon-precomposed" sizes="152x152" href="./img/components/icons/favicon-apple-152x152.png">
+  <script type="text/javascript">(function () { var PAGE_LOADED_TIME = Date.now(); window.getPageLoadedTime = function () { return PAGE_LOADED_TIME; } })();</script>
+  <!--[if BOOTSTRAP-CSS]><![endif]-->
```

Comment:
Related to my other question, is this block still needed?

Reply:
I have searched for bootstrap and did not find it, that is why I removed the StringReplacePlugin, might be that I was mistaken 🤔 But if we don't need it we can remove it here, too